trollflow2/plugins/__init__.py

```diff
@@ -413,7 +413,9 @@ def covers(job):
 
     product_list = job['product_list'].copy()
 
-    scn_mda = job['scene'].attrs.copy()
+    scn_mda = {"start_time": job['scene'].start_time,
+               "end_time": job['scene'].end_time,
+               "sensor": job['scene'].sensor_names}
```

Comment:
This is duplicated further down, should we have a small function for that?

Reply:
Done. Merging and releasing after the tests have passed.